app/Http/Controllers/PostController.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Post;
+use Illuminate\Http\Request;
+
+class PostController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $products = Post::latest()->paginate(5);
```

Comment:
remova a variável que não está sendo utilizada @GabrielaWolff 
```suggestion
        Post::latest()->paginate(5);
```